lms/static/scripts/postmessage_json_rpc/client/test/client-test.js

```diff
@@ -0,0 +1,159 @@
+import EventEmitter from 'tiny-emitter';
```

Comment:
I totally see that you brought this over from the client implementation, but I'm curious about this dependency. It seems like it exists to allow these tests to run in non-Node envs that don't have `events.EventEmitter`? But then, I'm like, when do we run these tests not in a Node environment/where we wouldn't have access to the `events` core module?

Reply:
cant answer that, so lets discuss next week, good q.